src/index.js

```diff
@@ -522,17 +521,29 @@ function createConfig(options, entry, format, writeMeta) {
 							],
 						],
 					}),
-					// Only used for async await
 					babel({
-						// We mainly use bublé to transpile JS and only use babel to
-						// transpile down `async/await`. To prevent conflicts with user
-						// supplied configurations we set this option to false. Note
-						// that we never supported using custom babel configs anyway.
-						babelrc: false,
 						extensions: EXTENSIONS,
 						exclude: 'node_modules/**',
+						passPerPreset: true, // @see https://babeljs.io/docs/en/options#passperpreset
+						presets: [
+							[
+								'@babel/preset-env',
+								{
+									loose: true,
+									modules: false,
+									targets:
+										options.target === 'node' ? { node: '8' } : undefined,
+									exclude: ['transform-async-to-generator'],
+								},
+							],
+						],
 						plugins: [
-							require.resolve('@babel/plugin-syntax-jsx'),
+							[
+								require.resolve('@babel/plugin-transform-react-jsx'),
+								{
+									pragma: options.jsx || 'h',
```

Comment:
We probably need to add `pragmaFrag` too to be compatible with `preact`. By default `pragmaFrag` will be transpiled to `React.Fragment`: https://babeljs.io/docs/en/babel-plugin-transform-react-jsx#fragments

Reply:
Good call 🤙 I'll do it and see if I can move ts and flow to babel and undraft

Reply:
still needs some ts checking and output the tsdefs 👀